starter-code/french-onion-soup-starter-code/css/main.css

```diff
@@ -1,9 +1,91 @@
-/*Some necessary adjustments*/
-h1, body, html, main {
+
+body, html, main {
   margin: 0;
 }
-/*Import the font and set the default size*/
+
 html {
   font-family: 'Open Sans', sans-serif;
   font-size: 18px
 }
+
+body{
+  font-weight: 100;
+}
+
+h1{
+  font-size: 45px;
+  font-weight: 100;
+}
+
+/* http://jsfiddle.net/qNjSp/ */
+ol {
+  counter-reset: list;
+}
+
+ol > li {
+  list-style: none;
+}
+
+ol > li:before {
+  content: counter(list) ")  ";
+  counter-increment: list;
+}
+/* http://jsfiddle.net/qNjSp/ */
+
+.hero{
+  height: 500px;
+  width: 100%;
+  position: relative;
+  text-align: center;
+}
+
+.hero-image{
+  background-image: url("../images/soup.jpg");
+  background-size: 100% 100%;
+  background-repeat: no-repeat;
+  height: 100%;
```

Comment:
you can play around with the width and height of image in order to get it the size you want.